packages/core/src/InstanceSettingsConfig.ts

```diff
@@ -0,0 +1,12 @@
+import { Config, Env } from '@n8n/config';
+
+@Config
+export class InstanceSettingsConfig {
```

Comment:
Not maybe the best name. Thought using `SecurityConfig` but if we move the other envs `InstanceSettings` uses (`N8N_USER_FOLDER` and `N8N_ENCRYPTION_KEY`) that name wouldn't make sense